src/applications/discharge-wizard/components/v2/ReviewPage.jsx

```diff
@@ -29,30 +39,49 @@ const ReviewPage = ({ formResponses, router, viewedIntroPage }) => {
   );
 
   const onEditAnswerClick = route => {
+    toggleEditMode(true);
     router.push(route);
   };
 
+  const onBackClick = () => {
+    const lastRoute = routeMap[routeMap.length - 2];
```

Comment:
Consider: should we use `previous` here instead of `last` since `last` can also indicate the (not in edit mode) last question in the flow before the review screen?